interfaces/builtin/screen_inhibit_control.go

```diff
@@ -58,16 +58,16 @@ dbus (send)
 # freedesktop.org ScreenSaver
 dbus (send)
     bus=session
-    path=/{,org/freedesktop/,org.gnome/}Screensaver
-    interface=org.freedesktop.ScreenSaver
-    member=org.freedesktop.ScreenSaver.{Inhibit,UnInhibit,SimulateUserActivity}
+    path=/{,org/freedesktop/,org.gnome/}Screen{s,S}aver
```

Comment:
I'm looking at GNOME 3.26, so it might be different than older versions, but GNOME guys seem to be consistent and expose `org.gnome.ScreenSaver` only at `/org/gnome/ScreenSaver` path and the interface does not have `.{Inhibit,UnInhibit,SimulateUserActivity}` methods.